### ISSUE TITLE
Support Macports

### ISSUE THREAD

**Issue by esafak** [raphamorim/rio#284]
Macports is a popular package manager for MacOS. I see that you support homebrew. Adding macports would increase your distribution.

**Comment by raphamorim** [comment]
Hey @esafak thanks for the issue.

I agree, do you know how the process works for add rio in macports? Is this something you would like to contribute?

**Comment by esafak** [comment]
Yes, I do. Unfortunately, I do not have the time and I barely use rio at the moment so this is more for your you and your users. You just need to create a PR on GitHub. [Here](https://github.com/macports/macports-ports/blob/master/www/blades/Portfile) is a cargo-based example you can copy. The salient parts are L5 (github.setup) and the checksums (L35-38).
